packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -862,6 +936,24 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
         this.update();
     }
 
+    private onKeyDown(event: _ModuleSupport.KeyInteractionEvent<'keydown'>) {
+        const { annotationData, state } = this;
+
+        const context = this.getAnnotationContext();
+        if (!context) return;
+
+        const datum = annotationData.at(-1);
+        const { key } = event.sourceEvent;
+
+        if (key === 'Tab') {
```

Comment:
There are other events that should trigger the saving of the text, will also look into this when doing the editing feature as it may be better to move some of this into the state machine instead.